package.json

```diff
@@ -82,6 +82,7 @@
     "nouislider": "^14.6.3",
     "rxjs": "^6.6.7",
     "stacktrace-js": "^2.0.2",
+    "subtitles-parser-vtt": "^0.1.0",
```

Comment:
As the subtitles-parser is only used by the scripts workspace we can move it to the package.json there instead (or add via `yarn workspace scripts add subtitles-parser-vtt`. 

This won't show any error because yarn also checks the top node-modules for packages (usual behaviour if for yarn to hoist node_modules from workspace folders to the top folder where possible to avoid package duplication, although I've disabled this as often leads to issues if one workspace relies on something like webpack 4 and the other webpack 5)

I'll quickly push the change directly.